docs/reference/ddl/acl.rst

```diff
@@ -0,0 +1,142 @@
+.. _ref_eql_ddl_acl:
```

Comment:
How do you feel about switching to use `ols` instead of `acl` in all the refs and filenames? I think that's a little more consistent with the messaging, since we seem to have converged on referring to this as "object-level security".

Reply:
I think we should keep using ACL as that's an established term that people will be searching for. OLS might be more accurate, but no one knows it and the benefit of switching to using it is tiny.

Reply:
Other than in the filenames and git commit messages I don't think I use the term ACL. The docs talk about "access policies" implementing "object-level security".